asv/repo.py

```diff
@@ -181,11 +181,11 @@ def get_repo(conf):
     if conf.dvcs is not None:
         for cls in util.iter_subclasses(Repo):
             if getattr(cls, 'dvcs') == conf.dvcs:
-                return cls(conf.repo, conf.project)
+                return cls(conf.repo, conf.mirror_dir)
```

Comment:
Yes, it's an API change. I thought that because the mirrored repository
in general is bare and there's no working tree, the name of the
directory is not important. It's of course possible to change it to
default to config.project if that's better.